index.bs

```diff
@@ -5842,7 +5842,7 @@ their computation, based on the <a>computedValue</a> of the
 The six coefficients (\(b_0, b_1, b_2, a_0, a_1, a_2\)) for each
 filter type, are:
 
-: {{"lowpass"}}
+: "{{lowpass}}"
```

Comment:
I'll let this thru, but it can go either way - if a WebIDL block is doing the dfn'ing of the term, it gets defined both with and without quotes.

Reply:
Hmm. bikeshed was complaining about this, and changing it made the link error go away.